src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_DirectToPage.js

```diff
@@ -23,10 +24,21 @@ class CJ4_FMC_DirectToPage {
 
         //Get total count of waypoints, including arrival and approach waypoints
         let approachWaypointsCount = fmc.flightPlanManager.getApproachWaypoints().length;
-        let waypointsCount = fmc.flightPlanManager.getWaypointsCount() + fmc.flightPlanManager.getArrivalWaypointsCount() + approachWaypointsCount;
+        let waypointsCount = fmc.flightPlanManager.getWaypointsCount() + approachWaypointsCount;
+        
+        //temporary
+        console.log("idx:" + fmc.flightPlanManager.getActiveWaypointIndex());
```

Comment:
Are we leaving the logging here for now?

Reply:
Yea, I want to leave it for now so that if I encounter any weirdness, I can check the log. Can pull later before it is not a beta release.